README.md

```diff
@@ -1 +1,78 @@
-# ocw-course-hugo-starter
\ No newline at end of file
+# ocw-course-hugo-starter
+
+A starter template for a Hugo site that uses [`ocw-course-hugo-theme`](https://github.com/mitodl/ocw-course-hugo-theme)
+
+![image](https://user-images.githubusercontent.com/12089658/102410491-9b7e1680-3fbe-11eb-9746-e5ae996add40.png)
+
+## Prerequisites
+
+ - NodeJS
+ - Hugo
+
+## Building JS & CSS
+
+To build the JS and CSS for the theme, make sure you have the above prerequisites installed and run the following:
+
+```sh
+./build.sh
+```
+
+## Running the site
+
+To preview your site, run the following command:
+
+```sh
+hugo server
+```
+
+Hugo will watch your site source and refresh the page as you make changes to the site content.
+
+## Adding content
+
+Course level metadata for your course should be placed in the JSON format under the `data/courses` folder with a filename matching the course's ID.  For example: `data/courses/18-01-single-variable-calculus-fall-2006.json`.
+
+### Course home page
+
+Your course's home page can be created by creating an `_index.md` file in the `content` folder.  Here is an example for `18-01-single-variable-calculus-fall-2006`:
+
+```md
+---
+title: Course Home
+type: course
+layout: course_home
+course_id: 18-01-single-variable-calculus-fall-2006
+menu:
+  18-01-single-variable-calculus-fall-2006:
+    identifier: course-home
+    weight: -10
+---
+This introductory calculus course covers differentiation and integration of functions of one variable, with applications.
+
+```
+
+### Course sections
+
+Course sections should be placed in `content/sections`.  Here is an example of the beginning of the file representing the Syllabus section of `18-01-single-variable-calculus-fall-2006`:
+
+```md
+---
+title: Syllabus
+course_id: 18-01-single-variable-calculus-fall-2006
+type: course
+layout: course_section
+menu:
+  18-01-single-variable-calculus-fall-2006:
+    identifier: 65cb45fb6cb01e83dce485f41e5471b5
+    weight: 10
+
+---
+## Course Meeting Times
+
+Lectures: 3 sessions / week, 1 hour / session
+
+Recitation: 2 sessions / week, 1 hour / session\[  
+\]({{% getpage "sections/video-lectures-creole/_index.md" %}})
+...
+```
+
+The `type` and `layout` values in front matter tell Hugo which page to render, and the `menu` object creates a nav entry for the page.  Detailed info on course-level metadata and layouts can be found in the [`ocw-course-hugo-theme`](https://github.com/mitodl/ocw-course-hugo-theme) readme.
```

Comment:
After creating the above files:

`content/_index.md`
`content/sections/meetings.md`
`data/courses/18-01-single-variable-calculus-fall-2006.json` (contains just `{}`}

and then running `hugo server`, I get this error:


> Error: Error building site: failed to render pages: render of "home" failed: "/Users/mbertrand/code/ocw-course-hugo-starter/_vendor/github.com/mitodl/ocw-course-hugo-theme/layouts/course/course_home.html:3:18": execute of template failed: template: course/course_home.html:3:5: executing "course/course_home.html" at <partial "head.html" .>: error calling partial: execute of template failed: template: partials/head.html:4:5: executing "partials/head.html" at <partial "title.html" .>: error calling partial: "/Users/mbertrand/code/ocw-course-hugo-starter/_vendor/github.com/mitodl/ocw-course-hugo-theme/layouts/partials/title.html:3:18": execute of template failed: template: partials/title.html:3:18: executing "partials/title.html" at <index .Site.Data.courses $courseId>: error calling index: index of untyped nil


Is this because I don't have required info in the json file?